docs/cli.md

```diff
@@ -147,31 +147,41 @@ For instance, for ES6 support with [Babel](https://babeljs.io/) 7 add:
 
 This will effectivally call `require('@babel/register')` prior to requiring any support files.
 
-### Non JS files
+### Typescript (and other non-JS files)
```

Comment:
I would suggest a separate `TypeScript` and `Other languages` section. In the  `TypeScript` section, just suggest ts-node and not babel.

Reply:
I've reorganized the whole transpilation chapter.

Reviewing only the diff in github miss some context. I suggest to review the whole "transpilation" chapter